src/main/java/io/epirus/console/account/AccountManager.java

```diff
@@ -53,59 +54,123 @@ public static void main(final CliConfig config, final String[] args) throws IOEx
         }
     }
 
-    public void createAccount(String email) throws IOException {
+    @VisibleForTesting
+    public AccountManager(final CliConfig cliConfig, OkHttpClient client) {
+        this.client = client;
+        this.config = cliConfig;
+        this.cloudURL = DEFAULT_CLOUD_URL;
+    }
+
+    @VisibleForTesting
+    public AccountManager(final CliConfig cliConfig, OkHttpClient client, String cloudURL) {
+        this.client = client;
+        this.config = cliConfig;
+        this.cloudURL = cloudURL;
+    }
+
+    public void createAccount(String email) {
         RequestBody requestBody = createRequestBody(email);
         Request newAccountRequest = createRequest(requestBody);
-
         try {
             Response sendRawResponse = executeClientCall(newAccountRequest);
             ResponseBody body;
             if (sendRawResponse.code() == 200 && (body = sendRawResponse.body()) != null) {
                 String rawResponse = body.string();
                 JsonObject responseJsonObj = JsonParser.parseString(rawResponse).getAsJsonObject();
-
                 if (responseJsonObj.get("token") == null) {
-
+                    System.out.println("Response token is null");
                     String tokenError = responseJsonObj.get("tokenError").getAsString();
                     if (tokenError == null || tokenError.isEmpty()) {
-                        System.out.println("Could not retrieve token. Try again later.");
+                        Console.exitError("Could not retrieve token. Try again later.");
                     } else {
-                        System.out.println(tokenError);
+
+                        Console.exitError(tokenError);
                     }
                     return;
                 }
                 String token = responseJsonObj.get("token").getAsString();
                 config.setLoginToken(token);
+                config.save();
```

Comment:
At some point we should hopefully be able to guarantee that config is loaded once when the CLI starts, and then saved once before the CLI exits, rather than saving at different times - no change necessarily needed here at the moment, but a good thing to discuss in the future.